app/src/main/java/org/oppia/android/app/survey/ExitSurveyConfirmationDialogFragmentPresenter.kt

```diff
@@ -57,7 +55,7 @@ class ExitSurveyConfirmationDialogFragmentPresenter @Inject constructor(
   }
 
   private fun endSurveyWithCallback(callback: () -> Unit) {
-    surveyController.stopSurveySession().toLiveData().observe(
+    surveyController.stopSurveySession(isCompletion = false).toLiveData().observe(
```

Comment:
Perhaps isCompletion --> surveyIsComplete or surveyCompleted

(again, trying to avoid starting booleans with a verb since such names are interpretable as functions)

Reply:
Noted, thanks. I have used both suggestions where I thought they would read more naturally.

Reply:
There are still some more instances of isCompletion in the PR, it might be worth fixing them too.

Reply:
Updated